wgsl/index.bs

```diff
@@ -8627,12 +8632,126 @@ where compatibility is defined by the following table.
 See the [$validating GPUProgrammableStage|WebGPU API$]
 specification for interface validation requirements.
 
+### Buffer Binding Determines Runtime-Sized Array Element Count ### {#buffer-binding-determines-runtime-sized-array-element-count}
```

Comment:
In WGSL the number of elements is available using the `arrayLength` built-in function. It seems remiss not to link to it in this section. Perhaps the description for arrayLength should also link to this section.

Reply:
Line 12622 updates the arrayLength description to point at NRuntime, whose definition is in the new section.

> It seems remiss not to link to it in this section.

Will add a link to arrayLength from this section.

Reply:
Fixed PTAL.